src/scienceDatabase.cpp

```diff
@@ -65,6 +99,14 @@ static string directionLabel(float direction)
     return name;
 }
 
+void flushDatabaseData() {
```

Comment:
code style, opening brackets on next line (I'm going to be a tiny bit more strict on this then before)